library/extensions/src/main/java/com/epmedu/animeal/extensions/Number.kt

```diff
@@ -0,0 +1,8 @@
+package com.epmedu.animeal.extensions
+
+fun Long.formatMetersToKilometers(): String {
```

Comment:
Looks like it won't work for Gerogian language

Reply:
Indeed, using string resources now